packages/google_maps_flutter/google_maps_flutter_web/README.md

```diff
@@ -10,8 +10,8 @@ This package is not an endorsed implementation of the google_maps_flutter plugin
 
 ```yaml
 dependencies:
-  google_maps_flutter: ^0.5.28
-  google_maps_flutter_web: ^0.1.0
+  google_maps_flutter: ^2.0.0
+  google_maps_flutter_web: ^0.3.0
```

Comment:
We should just remove this code section, and adjust the line above to say something like: `[...] so you'll need to [add it explicitly](https://pub.dev/packages/google_maps_flutter_web/install)`

We should get out of the business of having doomed-to-bitrot installation examples in our READMEs, and let pub.dev's dynamic page handle that.

Reply:
done